dev-tools/build-packages/rpm/Docker/Dockerfile

```diff
@@ -5,16 +5,17 @@ RUN yum clean all && yum update -y
 RUN yum install -y openssh-clients sudo gnupg \
     yum-utils epel-release redhat-rpm-config rpm-devel \
     zlib zlib-devel rpm-build autoconf automake \
-    glibc-devel libtool perl
+    glibc-devel libtool perl jq
 
-#RUN yum install -y --nobest https://repo.ius.io/ius-release-el7.rpm  
+#RUN yum install -y --nobest https://repo.ius.io/ius-release-el7.rpm
 
 RUN yum update -y && yum install -y python3
 
 RUN ln -fs /usr/bin/python3.6 /usr/bin/python
 
 # Add the scripts to build the RPM package
-ADD builder.sh /usr/local/bin/builder
+ADD rpm-builder.sh /usr/local/bin/builder
+ADD wazuh-dashboard.spec /usr/local/bin/wazuh-dashboard.spec
```

Comment:
**nitpick**: I understand this is a RPM specification file and it seems confusing to me the file is copied to a directory that should contain binaries `/usr/local/bin`.

Reply:
You are correct. Solved in 6a853002f3e67017b8c677eb41f3b6bd0eed0b78 (for `deb` too)